gsy_framework/constants_limits.py

```diff
@@ -247,6 +248,7 @@ class GlobalConfig:
     DURATION_D = 1
     MARKET_COUNT = 1
     FUTURE_MARKET_DURATION_HOURS = 0
+    DAY_AHEAD_DURATION_DAYS = 1
```

Comment:
Here I'm always not sure of the difference between `GlobalSettings` and `ConstSettings.FutureMarketSettings`. Could you remind me why we don't put both in the same class?

Reply:
Good question! I always forget. Having a look at the docstring does not really help either. 
I think I just copied the behaviour from the futures markets' FUTURE_MARKET_DURATION_HOURS. Which does not conform to the above comment `Default simulation settings d3a-web side` ( I will exchange d3a with gsy, too) either.
 
I propose to move both FUTURE_MARKET_DURATION_HOURS and FUTURE_MARKET_DURATION_HOURS into ConstSettings. FUTURE_MARKET_DURATION_HOURS. What do you think?

Reply:
That works for me 👍 Thanks!